site/content/en/references/kustomize/kustomization/replacements/_index.md

```diff
@@ -198,10 +198,14 @@ Strings are used for mapping nodes. For sequence nodes, we support two options:
 
```

Comment:
Above it says "For sequence nodes, we support two options". This should say three, not two.